EmaPajic/assignment1/assignment1_Q1.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+import sys
+
+def areAnagrams(str1,str2,isCaseSensitive):
```

Comment:
Can you please add a comment explaining your line of thought for the algorithm of this function?

Reply:
updated